src/sentry/models/userip.py

```diff
@@ -24,8 +27,22 @@ class Meta:
 
     @classmethod
     def log(cls, user, ip_address):
+        try:
+            geo = geo_by_addr(ip_address)
+        except Exception:
+            geo = None
+
+        values = {
+            'last_seen': timezone.now(),
+        }
+        if geo:
+            values.update({
+                'country_code': geo['country_code'],
+                'region_code': geo['region'],
+            })
```

Comment:
Would it be useful to track user-agents as well? Doing so would allow us to display a bit more context around device/browser type used.

Reply:
could be a good followup -- we dont do it anywhere in the security notices atm